### DISCUSSION TITLE
can server address

### DISCUSSION THREAD

**Discussion by artdelphi** [hazzelnuts/ntfy-for-delphi#35] [Q&A]
Can I change the server to my own?

**Comment by p-samuel** [comment]
@artdelphi , yes you can.  which platform are you targeting? If Windows, you'll must have Docker installed and run the commands shown on the readme file of the tests folder. This is everything you'll have to do for a simple local network. But if your goal is the web then the best solution setting up on a cloud linux machine. There are several websites that helps you to run a server on low budget like [Railways](https://railway.app/), [Vuttr](https://www.vultr.com) and [Linode](https://www.linode.com/pt/), not mentioning countless others. The process of setting up is described on ntfy documentation. I'm planning to make a video later describing how to configure on one of these options but I guess there is already some material in the internet.

**Reply by artdelphi** [reply]
thank you sir.

**Comment by p-samuel** [comment]
Frankly speaking, except the kind of information you are exchanging be very sensitive, I see no reason utilizing a server of your own since ntfy already has a free server available.

**Reply by artdelphi** [reply]
thank you sir.

**Comment by artdelphi** [comment]
thank you sir

Art Programmer Delphi

On 17 Feb BE 2566, at 18:16, Samuel ***@***.***> wrote:

﻿

Frankly speaking, excpet the kind of information you are exchanging is very sensitive, I see no why utilizing a server of your own since ntfy already has a free server available.

**Comment by artdelphi** [comment]
thank you sir

Art Programmer Delphi

On 17 Feb BE 2566, at 18:17, Samuel ***@***.***> wrote:

﻿

Frankly speaking, excpet the kind of information you are exchanging is very sensitive, I see no why utilizing a server of your own since ntfy already has a free server available.

**Comment by artdelphi** [comment]
thank you sir

Art Programmer Delphi

On 17 Feb BE 2566, at 18:12, Samuel ***@***.***> wrote:

﻿

@artdelphi<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fartdelphi&data=05%7C01%7C%7C9054053cb0c949f5c33c08db10d7ce07%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638122291288455717%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=7NF4UCc7Y6KtPJOcE2GQ1x7ENSXtT%2FHEoBuveJRKxmU%3D&reserved=0> , which platform are you targeting? If Windows, you'll must have Docker installed and run the commands shown on the readme file of the tests folder. This is everything you'll have to do for a simple local network. But if your goal is the web then the best solution setting up on a cloud linux machine. There are several websites that helps you to run a server on low buget like Railways<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Frailway.app%2F&data=05%7C01%7C%7C9054053cb0c949f5c33c08db10d7ce07%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638122291288455717%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=M7qiTuyiL%2Fk3p2PC%2FvepPcJYT5%2FsDp4nc0mwA7qqh1M%3D&reserved=0>, Vuttr<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.vultr.com%2F&data=05%7C01%7C%7C9054053cb0c949f5c33c08db10d7ce07%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638122291288455717%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=O4MpLRxND5JscD8gSpaZBbgn1Cr6eAySAYtLD1l20KI%3D&reserved=0> and Linode<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linode.com%2Fpt%2F&data=05%7C01%7C%7C9054053cb0c949f5c33c08db10d7ce07%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638122291288455717%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=he8Trj0QrtsyNx89aSBjJNN5sSzBtgYk1uRPk%2Fwec0o%3D&reserved=0>, not mentioning countless others. The process of setting is described ntfy documentation. I'm planning to make a video later describing how to configure on one of these options but I guess there is already some material in the internet.

**Comment by p-samuel** [comment]
@artdelphi, I've uploaded a video on the readme file. Check it out.
